datalad_catalog/tests/data/workflow_generated_meta_sub.json

```diff
@@ -0,0 +1,139 @@
+{
+    "authors": [
+        {
+            "email": "s.heunis@fz-juelich.de",
+            "name": "Stephan Heunis"
+        },
```

Comment:
That's correct, from the extraction process. Controlling the git config via the proposed method seems like the perfect solution. Do you know which fixture to use in this case?

Reply:
If I understood correctly, adding a `conftest.py` file which imports `setup_package` from `datalad.conftest` would automatically execute that function at the beginning of pytest runs (`setup_package` has some pytest decorator). This would mimic all the setup tweaks done by DataLad core, which include making a fake home directory and setting git config `user.name=DataLad Tester`, `user.email=test@example.com`.

There might be other ways of changing the git config (direct git call or maybe some environment tweaks), but I also understand that it's safer to run them after doing `setup_package`, so that user's configuration wouldn't get affected.

Reply:
I agree. Just tested this locally with Yarik's PR merged, and it behaves as expected.